### ISSUE TITLE
Where should custom actions be assessed in our guidance (or under WCAG?)

### ISSUE THREAD

**Issue by detlevhfischer** [w3c/matf#101]
This is a placeholder issue for the point raised by @jamieherrera in the MATF call on 2025-03-19  - that we do not know where to place issues with custom actions.

IMO Custom actions would not fall under 2.5.1 Pointer gestures since that SC looks at touch input without AT.

@JJdeGroot suggested 2.1.1 Keyboard as a possible place. But custom actions would typically be a lot more relevant for screenreader users since the availablity of custom actions sio announced when you focus an elements (if there are actions present) - so I am nit sure that is a good home either...
Keyboard user would need to guess that they are there - or how would they otherwise be discoverable?

@qbalsdon mentioned there are ways of activating custom actions with the keyboard - on iOS, I haven't found a way of doing that - are you talking Android? Is that only available when using keyboard and screenreader at the same time? Or does it require special key combinations other than arrow keys (in my experience so far, arrow keys did not work to select custom actions).

When testing custom actions based on EN 301 549, there is actually a fitting requirement for some issues I find:
> 11.5.2.11 List of available actions
> Where the software provides a user interface it shall, by using the services as described in clause 11.5.2.3, make a list of available actions that can be executed on a user interface element, programmatically determinable by assistive technologies.

and 
> 11.5.2.12 Execution of available actions
> Where permitted by security requirements, software that provides a user interface shall, by using the services as described in clause 11.5.2.3, allow the programmatic execution of the actions exposed according to clause 11.5.2.11 by assistive technologies.

I have had cases where actions were announced / could be focused that then did not work so that could be failed here under 11.5.2.12.
Another case has been functionality available via actions, but to get to an equivalent function with the keyboard, you'd have to call up a subview (one of the actions). But if that action is not defined as the default action, the keyboard user had no way of getting to the subview but instead acitvates the defaut custom action (whatever that is).

**Comment by JJdeGroot** [comment]
Custom actions are now added to 2.5.3

and there are sub issues to consider them in 1.3.1 and 4.1.2
#200 + #216
